doc/DOCUMENTATION.org

```diff
@@ -402,12 +402,52 @@ Example to install =llvm-mode= and =dts-mode=:
 (setq dotspacemacs-additional-packages '(llvm-mode dts-mode)
 #+END_SRC
 
-** Packages synchronization (Vundle like feature)
```

Comment:
I omitted mention of Vundle here, because I didn't know how to word it.
I just don't know what it is.